Dockerfile

```diff
@@ -10,10 +10,8 @@ RUN npm install
 
 COPY . ./
 
-# The echo command is necessary to work around a bug in docpad 6.79.4
-RUN echo | $(npm bin)/docpad generate --env static --silent 
-
+RUN npm run build
 
 FROM nginx:1.18.0-alpine
 
-COPY --from=builder /app/out /usr/share/nginx/html
+COPY --from=builder /app/dist /usr/share/nginx/html
```

Comment:
Could you add a .dockerignore file as well? 

https://github.com/fluid-project/fluidproject.org/blob/master/.dockerignore

Also, if you could use Node.js 14.14.0 that be great. It'll be the LTS version in a few weeks.

Reply:
Sure thing, @gtirloni!

Reply:
Resolved in 910d906.